Source/Classes/NantesLabel.swift

```diff
@@ -487,17 +500,17 @@ public extension NSAttributedString.Key {
         return textRect
     }
 
-    /// We're handling link touches elsewhere, so we want to do nothing if we end up on a link
+    /// If there's a link at the touch point, handle that in touchesEnded
+    /// If there's a label tapped block, don't forward touches, run the block
+    /// Otherwise forward touches on
     override open func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
-        guard let touch = touches.first,
-            let activeLink = link(at: touch.location(in: self)) else {
-                if labelTappedBlock == nil {
-                    super.touchesBegan(touches, with: event)
-                }
-                return
+        if let touch = touches.first, let activeLink = link(at: touch.location(in: self)) {
```

Comment:
I just noticed a potential issue here. In the case of the truncation token overlapping a link, the link will consume the click rather than the token, which seems wrong. 

![Screenshot 2019-03-19 at 11 13 06](https://user-images.githubusercontent.com/1429306/54601903-1ad06600-4a38-11e9-9525-74a1a8db426f.png)